messages/lightning.preview.site.md

```diff
@@ -1,38 +1,37 @@
 # summary
 
-Preview Experience Cloud Sites.
+Preview an Experience Cloud site locally and in real-time, without deploying it.
 
 # description
 
-In dev preview mode, you can edit local files and see these changes to your Lightning Web Components (LWC) within your site:
+In developer preview mode, see local changes to your site in a real-time preview that you don't have to deploy or manually refresh. To let you quickly iterate on your Lightning web components (LWCs) and pages, your site preview automatically refreshes when source code changes are detected.
 
-- Basic HTML and CSS edits
-- Importing new CSS
-- Javascript edits in-service component library
-- Javascript method changes in the LWC component that don't alter its public API.
+When you edit local files in developer preview mode, your LWCs reflect these changes within your {org name} org:
 
-Other local changes may require deployment to your org. However, changes made directly in your org (like modifying component properties) are immediately live and won't show in your local files until you retrieve them from the org.
+- Basic HTML and CSS edits
+- Importing new CSS-only LWCs
+- JS edits in-service component library
+- JS method changes that don't affect the LWC's public API
 
-# flags.name.summary
+To see any other local changes in your site, you have to deploy to your org. If you make changes directly in your org, (like saving new component properties,) they're immediately live on your site. To update your local version of the site with those changes, you have to retrieve them from your org.
 
-Specify the site name for preview.
+Use the appropriate topic to preview specific aspects of the development environment.
 
-# flags.name.description
+# flags.name.summary
 
-The site name needs to match the name of a site on the current org. Example: "B2C CodeCept"
+Name of the Experience Cloud site to preview. It has to match a site name from the current org.
 
-# examples
+# flags.target-org.summary
```

Comment:
@nrkruk  is this flag required for sf lightning preview site? I think it's required for sf lightning preview app.

Reply:
Yes but if you are already logged into an org with the CLI, it will pick up the default authenticated org. So its not like it has to be explicitly specified every time

Reply:
Hopefully, the summary of --target-org takes care of that scenario:

`Username or alias of the target org. Not required if the `target-org` configuration variable is already set.
`
If you log in to an org and specify --set-default, then the target-org config var is automatically set locally.